### PR TITLE
Move adaflo parameters into new section

### PR DESCRIPTION
**Pull request by peterrum**
This PR proposes to introduce a dedicated `adaflo` section within the `Navier-Stokes` section. Depends on https://github.com/kronbichler/adaflo/pull/6.

@mschreter What do you think?

### CODE REVIEW

File: include/meltpooldg/simulations/rising_bubble/rising_bubble.json [hunk 1]
```diff
@@ -42,16 +42,24 @@
...
-      "lin relative tolerance"       : "1",
-      "lin velocity preconditioner"  : "ilu scalar",
-      "lin pressure mass preconditioner" : "ilu",
-      "lin its before inner solvers" : "30"
+    "adaflo" : {
+      "Navier-Stokes" : {
+        "dimension"         : "2",
+        "global refinements" : "0",
+        "velocity degree"    : "2",
+        "viscosity"          : "0.01",
```
Comment (peterrum) [inline include/meltpooldg/simulations/rising_bubble/rising_bubble.json:50]: We should also revisit the purpose of these parameters. Are these adaflo specific? They don't seem so.

Comment (mschreter) [reply include/meltpooldg/simulations/rising_bubble/rising_bubble.json:50]: No, they are not. As we discussed, we should only accept adaflo parameters that are parsed within MeltPoolDG. I will introduce new sections in the following days.

**Comment by mschreter** [comment]
Looks good to me, thanks @peterrum .

File: include/meltpooldg/simulations/rising_bubble/rising_bubble.json (hunk 1 above)
Comment (peterrum) [reply include/meltpooldg/simulations/rising_bubble/rising_bubble.json:50]: I have pushed the above branch to our adaflo branch as long as it is not merged in the official branch.

**Pull request merged**